net/arp/arp_send.c

```diff
@@ -372,7 +372,9 @@ int arp_send(in_addr_t ipaddr)
       /* Increment the retry count */
 
       state.snd_retries++;
-      nerr("ERROR: arp_wait failed: %d\n", ret);
+      nerr("ERROR: arp_wait failed: %d, ipaddr: %lu.%lu.%lu.%lu\n", ret,
```

Comment:
I've updated the macro and added `uint8_t` cast as currently only `PRIu32` is the portable way to print.